src/structures/MessageEmbed.js

```diff
@@ -185,10 +185,29 @@ class MessageEmbed {
    * @param {boolean} [inline=false] Set the field to display inline
    * @returns {MessageEmbed}
    */
-  addBlankField(inline = false) {
+  addBlankField(inline) {
     return this.addField('\u200B', '\u200B', inline);
   }
 
+  /**
+   * Removes, replaces, and inserts fields in the embed (max 25).
+   * @param {number} index The index to start at
+   * @param {number} deleteCount The number of fields to remove
+   * @param {StringResolvable} [name] The name of the field
+   * @param {StringResolvable} [value] The value of the field
+   * @param {boolean} [inline=false] Set the field to display inline
+   * @returns {MessageEmbed}
+   */
+  spliceField(index, deleteCount, name, value, inline) {
+    if (name && this.fields.length - deleteCount >= 25) throw new RangeError('EMBED_FIELD_COUNT');
```

Comment:
We should not limit field count in the builder.  We do not do this anywhere else, and it introduces unnecessary opinion into the library.

Reply:
While I agree it's unnecessary opinion, that behavior isn't being introduced by this pr, and does exist in master already for the addField method of this class. My thought being, if that was merged, that's the behavior that was agreed on for this class. (since addField was added in #1529 over a year ago) So parity was made with this line.

Not opposed to removing the check from both places though.

Reply:
Oh, I didn't notice that we already do that.  Personally would be a fan of removing it, since we don't do any sanity checks like this elsewhere that I'm aware of.